services/azure-devops/azure-devops-artifacts.service.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { addv } = require('../text-formatters')
+const {
+  renderVersionBadge: renderNugetVersionBadge,
+} = require('../nuget/nuget-helpers')
+const { NotFound } = require('..')
+const AzureDevOpsBase = require('./azure-devops-base')
+const { keywords } = require('./azure-devops-helpers')
+
+const getPackagesSchema = Joi.object({
+  count: Joi.number().required(),
```

Comment:
As an FYI we have various helpers for schemas too like we do for formatters/renderers, `nonNegativeInteger` from `../validators` would be applicable here

Reply:
Thanks for the feedback @calebcartwright . I will address these suggestions in the next day or two